pages/common/fswatch.md

```diff
@@ -0,0 +1,15 @@
+# fswatch
+
+> A cross-platform file change monitor. 
+
+- Run a bash command on file creation, update or deletion:
+
+`fswatch -0 {{path_to_watch}} | xargs -0 {{bash_command}}`
```

Comment:
What sort of paths can be here ? If its only directories, I think we should go with `{{path/to/watch}}`